index.html

```diff
@@ -94,8 +94,10 @@ <h1 class="main_block_title">
 
     <section class="section">
 
+        <div class="section_wrap">
+
         <h2 class="section_title">What do we offer</h2>
-        <img class="line" src="images/line.png" alt="">
+
 
```

Comment:
You should try to use one code style. Add empty lines between tags only for some reasons

[Read more](https://www.w3schools.com/html/html5_syntax.asp)